src/app/InteractionModelDelegate.h

```diff
@@ -111,8 +110,7 @@ class InteractionModelDelegate
     /**
      * Notification that a Command Send has received an Invoke Command Response and fails to process a command data element in that
      * command response
-     * @param[in]  apCommandSender A current command sender which can identify the command sender to the consumer, particularly
-     * during multiple command interactions
+     * @param[in]  apCommandSender A current command sender which can identify the command sender to the consumer, particularly w*
```

Comment:
did this comment get chopped off by accident?

Reply:
yep, accidental chopped, fixed, thanks